common/src/main/java/earth/terrarium/cadmus/common/network/packets/clientbound/SyncClaimsPacket.java

```diff
@@ -0,0 +1,47 @@
+package earth.terrarium.cadmus.common.network.packets.clientbound;
+
+import com.teamresourceful.bytecodecs.base.ByteCodec;
+import com.teamresourceful.bytecodecs.base.object.ObjectByteCodec;
+import com.teamresourceful.bytecodecs.defaults.MapCodec;
+import com.teamresourceful.resourcefullib.common.bytecodecs.ExtraByteCodecs;
+import com.teamresourceful.resourcefullib.common.network.Packet;
+import com.teamresourceful.resourcefullib.common.network.base.ClientboundPacketType;
+import com.teamresourceful.resourcefullib.common.network.base.NetworkHandle;
+import com.teamresourceful.resourcefullib.common.network.base.PacketType;
+import com.teamresourceful.resourcefullib.common.network.defaults.CodecPacketType;
+import earth.terrarium.cadmus.Cadmus;
+import earth.terrarium.cadmus.api.claims.ClaimApi;
+import earth.terrarium.cadmus.client.CadmusClient;
+import it.unimi.dsi.fastutil.objects.Object2BooleanMap;
+import it.unimi.dsi.fastutil.objects.Object2BooleanOpenHashMap;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.world.level.ChunkPos;
+import net.minecraft.world.level.Level;
+
+import java.util.Map;
+import java.util.UUID;
+
+public record SyncClaimsPacket(
+    ResourceKey<Level> dimension,
+    Map<UUID, Object2BooleanMap<ChunkPos>> claims
+) implements Packet<SyncClaimsPacket> {
+
+    public static final ClientboundPacketType<SyncClaimsPacket> TYPE = CodecPacketType.Client.create(
+        Cadmus.id("sync_claims"),
+        ObjectByteCodec.create(
+            ExtraByteCodecs.resourceKey(Registries.DIMENSION).fieldOf(SyncClaimsPacket::dimension),
+            new MapCodec<>(ByteCodec.UUID,
```

Comment:
use `ByteCodec.mapOf` so that the underlying code can change without issues if needed